doc/source/howtos/virt_sandbox.rst

```diff
@@ -340,22 +420,23 @@ To confirm that the user has been set up correctly, you can try to log in:
 
    Type 'help;' or '\h' for help. Type '\c' to clear the current input statement.
 
-To provide read/write access to the cluster for ProxySQL, add this user on one
+To provide read/write access to the cluster for |proxysql|, add this user on one
 of the |PXC| nodes:
 
-.. code-block:: text
+.. code-block:: mysql
 
-   mysql> CREATE USER 'sbuser'@'192.168.70.64' IDENTIFIED BY 'sbpass';
-   Query OK, 0 rows affected (0.01 sec)
+   mysql> CREATE USER 'sbuser'@'192.168.70.74'
+   IDENTIFIED WITH mysql_native_password by 'sbpass';
 
-   mysql> GRANT ALL ON *.* TO 'sbuser'@'192.168.70.64';
-   Query OK, 0 rows affected (0.00 sec)
+   mysql> GRANT ALL ON *.* TO 'sbuser'@'192.168.70.74';
 
-Testing the cluster with sysbench
-=================================
 
-After you set up |PXC| in a sandbox, you can test it using
-`sysbench <https://launchpad.net/sysbench/>`_.
+Testing the cluster with the `sysbench` benchmark tool
+================================================================================
+
+After you set up |PXC| in your testing environment, you can test it using
+``sysbench``.
+
 This example shows how to do it with ``sysbench`` from the EPEL repository.
 
 1. Create a database and a user for ``sysbench``:
```

Comment:
We can remove user creation commands from here. It is already added in `Creating ProxySQL Client User`